src/test/java/seedu/address/testutil/EntryUtil.java

```diff
@@ -54,7 +54,18 @@ public static String getEntryDetails(EntryType type, Entry entry) {
     public static String getEditEntryDescriptorDetails(EditCommand.EditEntryDescriptor descriptor) {
         StringBuilder sb = new StringBuilder();
         descriptor.getType().ifPresent(
-                type -> sb.append(PREFIX_TYPE).append(type.getEntryType().toString()).append(" "));
+                type -> {
+                    switch (type.getEntryType()) {
+                    case EXPENDITURE:
+                        sb.append(PREFIX_TYPE).append(EntryType.ENTRY_TYPE_EXPENDITURE).append(" ");
+                        break;
+                    case INCOME:
+                        sb.append(PREFIX_TYPE).append(EntryType.ENTRY_TYPE_INCOME).append(" ");
+                        break;
+                    default:
+                        break;
+                    }
+                });
```

Comment:
rough fix as the to string method in entryType was changed to return expenditure/income instead of e/I instead

alternative solution: have another method to return e/I in entryType

Reply:
we can use this fix for now, since this test case is testing on the return output if I'm not wrong.